examples/next/pages/ui/components/in-app-messaging/override-component/index.page.tsx

```diff
@@ -49,8 +50,8 @@ function App() {
     [displayMessage]
   );
 
-  // display custom message component on initial render
-  useEffect(displayCustomBannerMessage, [displayCustomBannerMessage]);
+  // // display custom message component on initial render
+  // useEffect(displayCustomBannerMessage, [displayCustomBannerMessage]);
```

Comment:
Added test to check that banner is shown on click event. So commented out code that shows banner on load.

Reply:
Is this example used in the docs?

Reply:
yes

Reply:
Added back the useEffect block and updated tests.